source/locker.js

```diff
@@ -1,80 +1,84 @@
-class Locker {
-    constructor() {
-        this.items = {};
-        this.usingLocalStorage = false;
+export default class Locker {
+  constructor(session = false) {
```

Comment:
Accidentally reformatted! The only lines that changed though are in the constructor, and lines where we access `this.storage`